packages/terra-site/src/examples/table/NoStripedTable.jsx

```diff
@@ -0,0 +1,34 @@
+/* eslint-disable import/no-extraneous-dependencies */
+import React from 'react';
+import Table from 'terra-table';
+
+const shortid = require('shortid');
```

Comment:
Let's move this to the war room tomorrow. In my component, I'm finding a need to generate random keys, and so far, this is the best solution I found.